satpy/etc/composites/ahi.yaml

```diff
@@ -14,6 +14,19 @@ modifiers:
     - solar_azimuth_angle
     - solar_zenith_angle
 
+  true_color_reproduction_rayleigh_corrected:
```

Comment:
This is probably not specific to the true color reproduction composite, so could it be rename to eg: `no_aerosol_rayleigh_corrected` ?

Reply:
good idea!